src/amo/components/HomeHeroBanner/index.js

```diff
@@ -17,202 +18,18 @@ export class HomeHeroBannerBase extends React.Component<Props> {
-      </HeroSection>,
-      <HeroSection key="hero-19" linkTo="/addon/search-encrypt/">
-        <h3>{i18n.gettext('Search Encrypt')}</h3>
-
-        <p>{i18n.gettext('Privacy protection for your search data')}</p>
-      </HeroSection>,
-      <HeroSection key="hero-20" linkTo="/addon/auto-tab-discard/">
-        <h3>{i18n.gettext('Auto Tab Discard')}</h3>
-
-        <p>{i18n.gettext('Save memory by disabling inactive tabs')}</p>
-      </HeroSection>,
-      <HeroSection key="hero-21" linkTo="/addon/update-scanner/">
-        <h3>{i18n.gettext('Update Scanner')}</h3>
-
-        <p>
-          {i18n.gettext(
-            'Get notified when your selected websites update with new content',
-          )}
-        </p>
-      </HeroSection>,
-      <HeroSection key="hero-22" linkTo="/addon/autofullscreen/">
-        <h3>{i18n.gettext('Auto Fullscreen')}</h3>
-
-        <p>{i18n.gettext('Automatically start Firefox in full screen mode')}</p>
-      </HeroSection>,
-      <HeroSection key="hero-23" linkTo="/addon/videospeed/">
-        <h3>{i18n.gettext('Video Speed Controller')}</h3>
-
-        <p>{i18n.gettext('Adjust video playback speeds with shortcuts')}</p>
-      </HeroSection>,
-      <HeroSection key="hero-24" linkTo="/addon/view-image/">
-        <h3>{i18n.gettext('View Image')}</h3>
-
-        <p>
-          {i18n.gettext(
-            `Revive the ‘View Image’ and ‘Search by Image’ buttons on Google Images`,
-          )}
-        </p>
-      </HeroSection>,
-      <HeroSection key="hero-25" linkTo="/addon/neat-url/">
-        <h3>{i18n.gettext('Neat URL')}</h3>
-
-        <p>{i18n.gettext('Clean up links for easy sharing')}</p>
-      </HeroSection>,
-      <HeroSection key="hero-26" linkTo="/addon/glitterdrag/">
-        <h3>{i18n.gettext('Glitter Drag')}</h3>
-
-        <p>
-          {i18n.gettext(
-            'Drag text, images, or links to perform actions like copy, open, search, and more',
-          )}
-        </p>
-      </HeroSection>,
-      <HeroSection key="hero-27" linkTo="/addon/behind-the-overlay-revival/">
-        <h3>{i18n.gettext('Behind The Overlay Revival')}</h3>
-
-        <p>{i18n.gettext('Click a button to close annoying pop-ups')}</p>
-      </HeroSection>,
-      <HeroSection key="hero-28" linkTo="/addon/auto-sort-bookmarks/">
-        <h3>{i18n.gettext('Auto-Sort Bookmarks')}</h3>
-
-        <p>
-          {i18n.gettext(
-            'Automatically sorts bookmarks so they’re in your preferred position',
-          )}
-        </p>
-      </HeroSection>,
-      <HeroSection key="hero-29" linkTo="/addon/britannica-insights/">
-        <h3>{i18n.gettext('Britannica Insights')}</h3>
-
-        <p>{i18n.gettext('Add Britannica facts to your search results')}</p>
-      </HeroSection>,
-    ];
+    return heroes.map((hero, index) => {
+      const { description, title } = hero;
+      return (
+        /* eslint-disable react/no-array-index-key */
+        <HeroSection key={`hero-${index}`} linkTo={hero.url}>
```

Comment:
Using the array index as a key might cause some confusion during development when adding/removing items from the array (and hot reloading). What about using `hero.title` and `hero.url` as the key?